src/main/groovy/com/netgrif/application/engine/petrinet/domain/dataset/MultichoiceField.groovy

```diff
@@ -8,14 +8,14 @@ class MultichoiceField extends ChoiceField<Set<I18nString>> {
 
     MultichoiceField() {
         super()
-        super.setValue(new HashSet<I18nString>())
-        super.setDefaultValue(new HashSet<I18nString>())
+        super.setValue(new LinkedHashSet<I18nString>())
```

Comment:
Maybe it would be better to change definition of this class to `extends ChoiceField<LinkedHashSet<I18nString>>` so that it could not have any other implementation of `Set` set as value

Reply:
Updated definition